pkg/services/live/pushhttp/push.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/grafana/grafana/pkg/services/live/convert"
 	"github.com/grafana/grafana/pkg/services/live/pushurl"
 	"github.com/grafana/grafana/pkg/setting"
+
+	liveDto "github.com/grafana/grafana-plugin-sdk-go/live"
```

Comment:
why DTO?

Reply:
Already import live package `github.com/grafana/grafana/pkg/services/live` above, so there is a package name conflict. 

If you mean dto name - then comes mostly from this https://en.wikipedia.org/wiki/Data_transfer_object, specifically this part:

```
DTOs are simple objects that should not contain any business logic but may contain serialization and deserialization mechanisms for transferring data over the wire
```

The naming comes from my previous background, can rename if it's not very suitable from native speaker perspective.